Jenkinsfile

```diff
@@ -47,7 +49,7 @@ pipeline {
         disableConcurrentBuilds()
         timestamps()
         timeout(time: 3, unit: 'HOURS')
-        skipDefaultCheckout(false)
+        skipDefaultCheckout(true)
```

Comment:
Why do you provide a custom checkout?

Reply:
it was for deleting workspace purpose. not needed anymore i suppose.

Reply:
By my understanding, this related to the source checkout step. We could use the default one, or implement our own way to do it. You could see that the default was disabled and we must implement the "Checkout" stage instead of it. It's OK if it was done on purpose.